applications/vessel-history/src/features/settings/Settings.tsx

```diff
@@ -59,7 +70,10 @@ const Settings: React.FC = (): React.ReactElement => {
           onClick={onBackClick}
           className={styles.backButton}
         />
-        <h1>{t('settings.title',"Settings")} {selectedOption && <span> - {optionsData[selectedOption].title}</span>}</h1>
+        <h1>
+          {t('settings.title', 'Settings')}{' '}
```

Comment:
{' '} react not understand a simple space? :hankey:

Reply:
🤦 This was fixed by the auto-formatter on save 🤷‍♂️